packages/mock/src/faker/getters/object.ts

```diff
@@ -91,6 +91,7 @@ export const getMockObject = ({
       context,
       imports,
       existingReferencedProperties,
+      splitMockImplementations,
```

Comment:
This has nothing to do with the error, it was just missing and gave a syntax error